cloud-data-protection/src/components/login.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from "react";
+import {useDispatch} from "react-redux";
+import { login } from '../features/userSlice';
+import './login.css';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import "./login.css";
```